client/src/modules/templates/bhUserSelect.tmpl.html

```diff
@@ -0,0 +1,24 @@
+<div ng-form="{{ $ctrl.name }}" bh-user-select ng-model-options="{ updateOn: 'default' }">
```

Comment:
The `$ctrl.name` isn't passed in.  It would be best then to name this "UserForm"